### PR TITLE
Clean up email report script

### PR DESCRIPTION
**Pull request by gctucker**
Simplify the `email_report.py` code in preparation for adding support for results from test groups:

* Remove unnecessary exception handling
* Use `_` for all private methods
* Use object attributes when applicable
* Always print the email content on stdout

### CODE REVIEW

File: src/test_report.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def __init__(self, configs, args):
         self._email_user = os.getenv('EMAIL_USER')
         self._email_pass = os.getenv('EMAIL_PASSWORD')
 
-    def create_email(self, email_content, send_to, send_from, email_subject):
+    def _create_email(self, send_to, send_from, email_content, email_subject):
```
Comment (mgalka) [inline src/test_report.py:43]: It seems a bit more intuitive to me to have `email_subject` before `email_content`. It's nothing big and I may be biased. So it's rather "for consideration" than something crucial. I wouldn't probably even put a remark here if you haven't been changing fucntion's signature.

**Review by mgalka — COMMENTED** [review]
minor comments

File: src/test_report.py (hunk 1 above)
Comment (gctucker) [reply src/test_report.py:43]: Yes, my main reason for this change was to get the send to/from together and the email subject/content together as well.  But that's a fair point.

File: src/test_report.py [hunk 2]
```diff
@@ -131,19 +105,19 @@ def run(self):
...
-                                              self._email_send_to,
-                                              self._email_send_from,
-                                              email_sub)
-                email_server = self.smtp_connect(self._email_user,
-                                                 self._email_pass)
-                if email_server:
-                    self.send_mail(email_msg, email_server)
-                    email_server.quit()
+                print(content)
+                sys.stdout.flush()
```
[the remark this answers is not in the document]
Comment (gctucker) [reply src/test_report.py:112]: Nice, sometimes I should read the docs to learn what happened after Python 2.6.

**Pull request merged**